src/Ombi/ClientApp/src/styles/Styles.scss

```
@@ -1,181 +1,174 @@
+}
+
+.mat-autocomplete-panel{
+    background: $ombi-background-accent;
+}
+
+.mat-form-field.mat-focused .mat-form-field-ripple{
+    background-color: $ombi-active;
+}
+
+@media (hover: none){
+    .mat-expansion-panel:not(.mat-expanded):not([aria-disabled=true]) .mat-expansion-panel-header:hover{
```

Stylelint (17.3.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)